tests/phpunit/tests/multisite/site.php

```diff
@@ -2123,8 +2123,10 @@ public function test_wpmu_create_blog_cache_cleanup_backward_compatible() {
 
 			$blog_id = wpmu_create_blog( 'testsite1.example.org', '/test', 'test', 1, array( 'public' => 1 ), 2 );
 
-			// Should not hit blog_details cache initialised in $this->populate_options_callback tirggered during
-			// populate_options callback's call of get_blog_details.
+			/*
+			 * Should not hit blog_details cache initialized in $this->populate_options_callback triggered during
+			 * populate_options callback's call of get_blog_details.
+			 */
```

Comment:
The change related to the PR is fine, but just wanted to note that this comment broke my brain.

Reply:
SAME.